realsense2_camera/package.xml

```diff
@@ -19,6 +19,8 @@
   <depend>image_transport</depend>
   <depend>librealsense2</depend>
   <depend>rclcpp</depend>
+  <depend>rclcpp_lifecycle</depend>
```

Comment:
Depend on the parameter?
You know what, I am not sure.
Let's speak about it and decide together.
You way had an advantage that our Debians will also support lifecycle nodes which is great